### DISCUSSION TITLE
403 Forbidden after deployment

### DISCUSSION THREAD

**Discussion by dryant** [spatie/laravel-permission#2597] [Q&A]
Hi! 
My project works perfect on my machine, but when I deploy my project it's no possible navigate to control panel.
My project has been deployed on my VPS with plesk.
I have used Laravel Toolkit for deploy from github.
All databases have the same data than my local databasaes.
Remote database contains all spatie tables: roles, permissions, roles-has-permissions, and user-has-roles.
I have tried many things and I don't know what else to do.
My repository project is https://github.com/dryant/makezings (public repository)
How I can solve it?
Thanks!

**Comment by drbyte** [comment]
Staging your project on my localhost also works fine.
I installed it, created a database, copied .env.example to .env, and `php artisan key:generate`. Then `php artisan migrate --seed`.

I can visit the Filament `/cpanel` panel and login with `admin` and `alvaro` email addresses, and all the Filament resources display data without any `403` errors.

**Comment by drbyte** [comment]
I wonder if your deployed server might not have cache configured correctly? Perhaps you could explore customizing the `.env` settings related to cache.

**Comment by drbyte** [comment]
You said:
> All databases have the same data than my local databasaes.

... so I'm assuming you ran `php artisan db:seed` ... is that correct?

**Comment by drbyte** [comment]
If you use Tinker to query the system, does it give expected responses?

```php
$user = User::first();
// should show the default admin user

$user->roles;
// should show that it has 2 roles

```

These are based on the data in the seeders, as installed when calling `php artisan db:seed`

**Comment by dryant** [comment]
Hi! Thanks for your replies.

My MEMCACHED_HOST=127.0.0.1

Is correct?
